charts/kargo/templates/api/cluster-role.yaml

```diff
@@ -23,6 +23,13 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - create
+      - patch
```

Comment:
I am curious about the permission to patch Events. Aren't we only _creating_ events?

Same question goes for the controller.